keda/templates/10-keda-clusterrole.yaml

```diff
@@ -26,6 +26,7 @@ rules:
   - pods
   - secrets
   - services
+  - serviceaccounts
```

Comment:
@zroubalik I presume we'll need to backport this to https://github.com/kedacore/keda/blob/main/config/rbac/role.yaml#L23?

If so, I'll send a PR